src/settings/SettingsPage.js

```diff
@@ -11,6 +11,7 @@ import LanguageSelector from '../components/language-selector/LanguageSelector'
 import AnalyticsToggle from '../components/analytics-toggle/AnalyticsToggle'
 import JsonEditor from './editor/JsonEditor'
 import DesktopSettings from './DesktopSettings'
+import Experiments from '../components/experiments/experimentsPanel'
```

Comment:
nitpick: our component file names are `UpperCamelCased.js`

Reply:
Oops yep, I'll patch that up